cod.md

```diff
@@ -6,8 +6,69 @@ isStaticPost: true
 ---
 
 DevFest Vienna is dedicated to being open to everyone and making the conference a safe place for all. We do not discriminate based on ethnic background, religion, gender, sexuality, body shape, or disability. We do not tolerate any harassment.
+Actions stronger than a warning will be taken at the discretion of two or more members of the DevFest organizer team.
+
+False reports will be treated harshly. Personal disputes should be handled privately, unless actual harassment occurs.
+
+_DevFest Vienna reserves the right to remove any participant, sponsor or attendee at any time for any reason._
+
+This policy covers all organized events for the DevFest Vienna, including social events.
```

Comment:
I think this sentence is now appearing twice - here and at the bottom, or?

Reply:
I don't think so:
https://github.com/DevFestVienna/devfest-at-site/blob/54704ded44fe66a2fc4cd2a74e12cfc593a133a1/cod.md

Reply:
https://github.com/DevFestVienna/devfest-at-site/blob/a1f4d493513e7211319441a5841d9c254ee24be5/cod.md

This is the one which is referenced in the PR - when clicking on "View File" on the right.

Let me check, I'll make a merge on my Desktop and see what's happening :slightly_smiling_face:

Reply:
Yes, if I follow the instructions the text appears then twice.

Reply:
Huh! Super odd. I'll remove it locally, just in case